lab-jonathan/README.md

```diff
@@ -0,0 +1 @@
+This project is an http server, with a GET and a POST request.  I'm pulling out/parsing the query/message from the head on the GET and the body on the POST and running through Cowsay to reflect to the screen.  
```

Comment:
I'd like to see this extended a bit to go into more detail about the differing routes, and options that you can pass to your route with query strings.